buildtest/cli/report.py

```diff
@@ -874,29 +910,22 @@ def print_report_summary_output(
         color (str): An instance of a string class that tells print_report_summary what color the output should be printed in.
         detailed (bool, optional): Print detailed output of the report summary if ``buildtest report summary --detailed`` is specified
     """
-
-    console.print("Report File: ", report.reportfile())
-    console.print("Total Tests:", len(report.get_testids()))
-    console.print("Total Tests by Names: ", len(report.get_names()))
-    console.print("Number of buildspecs in report: ", len(report.get_buildspecs()))
+    consoleColor = checkColor(color)
+
+    console.print("Report File: ", report.reportfile(), style=consoleColor)
+    console.print("Total Tests:", len(report.get_testids()), style=consoleColor)
+    console.print("Total Tests by Names: ", len(report.get_names()), style=consoleColor)
+    console.print(
+        "Number of buildspecs in report: ",
+        len(report.get_buildspecs()),
+        style=consoleColor,
+    )
```

Comment:
for the time being let's not add `style=consoleColor` to each of these statements let it print out in the normal output whatever color that may be. However we do want to color the output of the tables for pass and fail tables so that looks right.

Reply:
console color removed